source/less/blocks/form-group.less

```
@@ -24,75 +218,250 @@
     margin-right: 14px;
   }
 
-  .group-input__name--contacts {
-    display: block;
+  .group-input__wrapper--radio {
+    margin-bottom: 51px;
+  }
+
+  .group-input__contacts {
+    padding-left: @padding-tablet;
+    padding-right: @padding-tablet;
     text-align: left;
-    padding-left: 22px;
-    padding-bottom: 8px;
-    padding-top: 12px;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-end;
   }
-  .group-input__input {
+
+  .group-input__heading--contacts {
+    text-align: center;
+    margin-bottom: 19px;
+    margin-top: 30px;
     position: relative;
-  }
 
-  .group-input__icon-phone {
-    background-color: @color-light-grey;
+    &::before {
+      content: "";
+      display: block;
+      position: absolute;
+      top: 18px;
+      left: -182px;
```

```suggestion
      left: 0;
```

И так же с другим псевдоэлементом (`right: 0;`)
А вот у самого `legent` ширину 100% и `text-align: center;`

И вообще так у всех `group-input__heading` с бордерами - общее правило (1 модификатор - 1 общее правило сразу на несколько заголовков)